src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/OriginalAlignment.java

```diff
@@ -0,0 +1,73 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypeBuilder;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFFormatHeaderLine;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.tools.AddOriginalAlignmentTags;
+import org.broadinstitute.hellbender.utils.*;
+import org.broadinstitute.hellbender.utils.genotyper.ReadLikelihoods;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
+import org.broadinstitute.hellbender.utils.logging.OneShotLogger;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFHeaderLines;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Original Alignment annotation counts the number of alt reads where the original alignment contig doesn't match the current alignment contig
+ *
+ * <p>If reads were realigned to multiple references (for example the full human reference followed by just the
+ * mitochondria) and the original alignment tag was recorded before realigning, then we can count the number of alt
+ * reads that have been realigned from other contigs to this one. This can be useful for downstream filtering if an alt
+ * allele has all or most of its support originally from a different contig. In the mitochondria case this can be useful
+ * for filtering known NuMTs that are present in other contigs in the reference.  </p>
+ *
+ * <h3>Caveat</h3>
+ * <p>This annotation can only be calculated if an OA tag is present in the bam and can only be run by Mutect2.</p>
```

Comment:
It will emit a warning and not be calculated because it doesn't have a LOD field (which is used to grab the best allele). We could probably change that to be more general, but I'm tempted to wait until there is a need.

Reply:
I was thinking about a check we used to have that would do an instanceof to find out what tool was calling the annotation, but that was more useful before we had the OneShotLogger.